msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,20 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot in the following scenarios:
+
+1. When the bot, itself, is installed and added to a conversation
+
+   > In team context, the activity's conversation.id is set to the `id` of the channel selected by the user during app installation or the channel where the bot was installed from.
+
+2. When a user is added to a conversation where the bot is installed
+
+   > User ids received in the event payload are unique to the bot and can be cached for future use, such as directly messaging a user.
+
+To determine if the new member added was the bot itself or a user, check the `Activity` object of the `turnContext`. If the `id` field of the `MembersAdded` object is the same as the `id` field of the `Recipient` object, then the member added is the bot, else it's a user. The bot's `id` generally is `28:<MicrosoftAppId>`.
```

Comment:
> field of the `MembersAdded` object i
`id` field of an element in the `membersAdded` list?

@vule-msft membersAdded is a list, right?

Reply:
Yes, thanks for catching @aosolis. This section got copied up from team member remove section during one of the iterations. I left it for consistency between the two sections in my last comment.

We should get this updated both sections as membersRemoved is also a list

@v-abirade let's update this line 
```If the `id` field of the `MembersAdded` object is the same as the `id` field of the `Recipient` object, then the member added is the bot, else it's a user.```

to

> If the `MembersAdded` list contains an object where `id` is the same as the `id` field of the `Recipient` object, then the member added is the bot, else it's a user.